src/py42/clients/alerts.py

```diff
@@ -104,3 +104,14 @@ def update_note(self, alert_id, note):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.update_note(alert_id, note)
+
+    def get_aggregate_data(self, alert_id):
+        """Get alert summary with details about observations.
```

Comment:
the other docs use the Present Simple tense for verbs, i.e. `Gets` instead of `Get`